core/src/test/resources/schema/postgres/nested-partitions-schema.sql

```diff
@@ -64,3 +64,62 @@ CREATE TABLE IF NOT EXISTS public.snapshot
     metadata        jsonb  NOT NULL,
     PRIMARY KEY (persistence_id, sequence_number)
 );
+
+DROP TRIGGER IF EXISTS trig_check_persistence_id_max_sequence_number ON public.journal_persistence_ids;
+DROP FUNCTION IF EXISTS public.check_persistence_id_max_sequence_number();
+DROP TRIGGER IF EXISTS trig_update_journal_persistence_ids ON public.journal;
+DROP FUNCTION IF EXISTS public.update_journal_persistence_ids();
+DROP TABLE IF EXISTS public.journal_persistence_ids;
+
+CREATE TABLE public.journal_persistence_ids(
+  id BIGSERIAL,
```

Comment:
suggestion: use `BIGINT GENERATED ALWAYS AS IDENTITY` as this is now the recommend way I believe.